docs/management_api.yml

```diff
@@ -527,6 +527,74 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /artifacts/generate:
+    post:
+      summary: Upload raw data to generate a new artifact
+      description: |
+        Generate a new Mender artifact from raw data and meta data. Multipart request with meta and raw file.
+        Supports generating single-file updates only, using the single file upload module. [https://hub.mender.io/t/single-file/486]: 
+      consumes:
+        - multipart/form-data
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          format: Bearer [token]
+          description: Contains the JWT token issued by the User Administration and Authentication Service.
+        - name: name
+          in: formData
```

Comment:
Same as above; is this correct by specification?

Reply:
Same as above, it's congruent with what we already have (POST /artifacts).